tests/autoyast/clone.pm

```diff
@@ -18,6 +18,8 @@ use utils qw(zypper_call);
 sub run {
     my $self = shift;
     assert_script_run 'rm -f /root/autoinst.xml';
+    assert_script_run 'cd /etc/zypp/repos.d';
+    assert_script_run "sed -i 's/enabled=1/enabled=0/' \$(ls|grep -i nvidia)";
```

Comment:
does it fail when cloning? can I see the run?
we don't plan to use the system afterwards, so if a repo that doesn't exist is not there is ok.

Reply:
Failed at clone test module: https://openqa.suse.de/tests/9426602#step/clone/3

Reply:
but then we should do this in the AutoYaST profile as you were doing at the begining? otherwise any module schedule would fail unless that code is there. is it possible do the post_script in autoyast after having clicking ok on the error dialog?

Reply:
I did use post_script, it also doesn't work. You can see: https://openqa.suse.de/tests/9426602/file/autoyast_home_encrypted.xml
I think it cannot disable NVIDIA at first, then clone it will include this NVIDIA repo.

Reply:
we should give a soft-failure here pointing to a bug (no answer from developers yet, so better create the bug). But we should soft-fails always, only if nvidia is found, that `ls|grep -i nvidia` should serve us as condition, could you please reshape it a bit, then should be ready to merge IMO.

Reply:
Use 'init-script' can fix this problem.